insight.go

```diff
@@ -25,6 +25,7 @@ type BasicInsightResponse struct {
 	CountryPrefix             string `json:"country_prefix,omitempty"`
 }
 
+// Format an error if we got one
```

Comment:
This didn't really need a godoc comment, as it's a private function :)

Reply:
And actually, "format an error" isn't quite accurate - although it uses `fmt.Errorf`, that's really a way to generate a new error object with the error message in a specific format. A more accurate description would be "Return an error describing what went wrong, if it did, otherwise `nil`" (although that wording's rubbish)